doc-content/enterprise-only/springboot/creating-self-contained-image-proc.adoc

```diff
@@ -108,9 +125,50 @@ The following example adds two version of the `Evaluation` artifact:
     <plugins>
   <build>
 ----
-. To build the self-contained Spring Boot image, enter the following command in the `service` directory:
+ifdef::PAM[]
+. Optional: if you want to be able to configure the {KIE_SERVER} to communicate with a {CENTRAL} Monitoring instance, add the following lines to the `pom.xml` file under the `<dependencies>` tag:
++
+[source, xml]
+----
+<dependency>
```

Comment:
this is if you want to comunicate Kie server to Business central through websockets.

Reply:
@elguardian is there any other way of communication of the Kie Server to Business Central, for monitoring purposes?